1주차/1316.py

```diff
@@ -0,0 +1,19 @@
+def solution(N):
+    group = 0  # 그룹 단어의 개수
+
+    for i in range(N):
```

Comment:
```suggestion
    for _ in range(N):
```
아래 for문에서도 i를 써서 헷갈릴 수 있으니까 다르게 써주는게 좋아요.
보통 이렇게 range의 원소를 꺼내 쓸 생각 없이 단순히 N번 반복만 하려는 for문같은 경우에는 _를 많이 써요